plugins/plugin-java/che-plugin-java-ext-lang-server/src/main/java/org/eclipse/che/plugin/java/server/rest/ClasspathService.java

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.java.server.rest;
+
+import org.eclipse.che.dto.server.DtoFactory;
+import org.eclipse.che.ide.ext.java.shared.dto.classpath.ClasspathEntry;
+import org.eclipse.jdt.core.IClasspathEntry;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.JavaModelException;
+import org.eclipse.jdt.internal.core.JavaModel;
+import org.eclipse.jdt.internal.core.JavaModelManager;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+
+/**
+ * Service for getting information about classpath.
+ *
+ * @author Valeriy Svydenko
+ */
+@Path("jdt/{wsId}/classpath/")
+public class ClasspathService {
+
+    private static final JavaModel model = JavaModelManager.getJavaModelManager().getJavaModel();
+
+    /**
+     * Returns information about classpath.
+     *
+     * @param projectPath
+     *         path to the current project
+     * @return list of classpath entries
+     * @throws JavaModelException
+     *         when JavaModel has a failure
+     */
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<ClasspathEntry> getClasspath(@QueryParam("projectpath") String projectPath) throws JavaModelException {
+        IJavaProject javaProject = model.getJavaProject(projectPath);
+        return convertToClasspathEntryDTO(javaProject.getRawClasspath());
+    }
+
+    private List<ClasspathEntry> convertToClasspathEntryDTO(IClasspathEntry[] entries) {
+        if (entries.length == 0) {
```

Comment:
this private method convertToClasspathEntryDTO is redundant. Please inline.
If ClasspathEntry is DTO it should contains DTO in class name